src/Sulu/Bundle/RouteBundle/Resources/config/doctrine/BaseRoute.orm.xml

```diff
@@ -19,6 +19,11 @@
                 <option name="default">1970-01-01 00:00:00</option>
             </options>
         </field>
+        <field name="changed" type="datetime" nullable="false">
```

Comment:
that's not possible because if you add the column MySQL throws an exception that this is not possible because of existing rows. we could write an SQL which first add this default values and then remove it afterwards. but that's quite hacky?